src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import Card from "./components/Card"
+import data from "./data"
+
+const App = () => {
+
+  const handleClick = () => window.alert("Clicked!")
+
+
+  const genLinks = () =>{
+    let arr = []
+    for(let i = 0; i< 10; i++){
+      arr.push({text : `Exercise ${i}`, link: `/exercise${i}`})
+    }
+    return arr;
+  }
+
+  const array = genLinks()
+  return (
+    <div className="h-screen flex flex-col items-center ">
+      <div>
+        {array.map((item, index)=>(
+          <a key={index} href={item.link} className={`px-2 underline underline-offset-4 ${!index == 0 && "border-indigo-600 border-l"} text-indigo-600 `} >{index == 0 ? "Home" : item.text}</a>
+        ))}
+      </div>
+      <div className="w-[40rem] p-5 h-72 mt-20 bg-[#dff8db] flex gap-5 items-center rounded-md">
+        {data.map(item=>(
+          <Card key={item.id} setup={item.setup} punchline={item.punchline} />
```

Comment:
⚠️ This is a personal preference, am not trying to impose it on you, but for me instead of passing many props into the component, I just use the spread operator and I spread the item that I am mapping, I think it makes the code looks cleaner and more readable, so here there is nothing right or wrong... you can think about it and you choose whatever you want.

```suggestion
          <Card key={item.id} {...item} />
```